tests/additional_seeds/additional_seeds_suite_test.go

```diff
@@ -0,0 +1,282 @@
+	for _, node := range info.Nodes {
+		if node.Ready && node.Started {
+			k := kubectl.ExecOnPod(node.Name, "--", "nodetool", "getseeds")
+			output := ns.OutputPanic(k)
+			seeds := re.FindAllString(output, -1)
+			if node.Seed {
+				seeds = append(seeds, node.IP)
+			}
+			sort.Strings(seeds)
+
+			Expect(seeds).To(Equal(expectedSeeds), "Expected pod %s to have seeds %v but had %v", node.Name, expectedSeeds, seeds)
+		}
+	}
+}
+
+func checkSeedConstraints() {
+	info := retrieveDatacenterInfo()
+	// There should be 3 seed nodes for every datacenter
+	checkThereAreAtLeastThreeSeedsPerDc(info)
+
+	// There should be 2 seed nodes per rack
+	// this is because of the additional seeds
+	checkThereIsAtLeastOneSeedNodePerRack(info)
+
+	// Seed nodes should not be down
+	checkDesignatedSeedNodesAreStartedAndReady(info)
+
+	// Ensure seed lists actually align
+	//
+	// NOTE: The following check does not presently work due to
+	// the lag time between when we update a seed label and when
+	// that change is reflected in DNS. Since we reload seed lists
+	// right after upating the label, some cassandra nodes will
+	// likely end up with slight out-of-date seed lists. KO-375
+	//
+	// checkCassandraSeedListsAlignWithSeedLabels(info)
+}
+
+func checkAdditionalSeedService() {
+	// Check the service
+
+	k := kubectl.Get(additionalSeedServiceResource).FormatOutput("json")
+	output := ns.OutputPanic(k)
+	data := map[string]interface{}{}
+	err := json.Unmarshal([]byte(output), &data)
+	Expect(err).ToNot(HaveOccurred())
+
+	err = json.Unmarshal([]byte(output), &data)
+
+	spec := data["spec"].(map[string]interface{})
+	actualType := spec["type"].(string)
+	Expect(actualType).To(Equal("ClusterIP"), "Expected additional seed service type %s to be ClusterIP", actualType)
+
+	// Check the endpoint
+
+	jsonpath := "jsonpath=\"{.subsets[0].addresses[0].ip}\""
+	k = kubectl.Get(additionalSeedEndpointResource).FormatOutput(jsonpath)
+	output = ns.OutputPanic(k)
+	actualIp := ""
+	err = json.Unmarshal([]byte(output), &actualIp)
+	Expect(err).ToNot(HaveOccurred())
+
+	err = json.Unmarshal([]byte(output), &actualIp)
+	Expect(actualIp).To(Equal("192.168.1.1"), "Expected additional seed endpoints IP %s to be 192.168.1.1", actualIp)
+}
+
+var _ = Describe(testName, func() {
+	Context("when in a new cluster", func() {
+		Specify("the operator can properly create an additional seed service", func() {
+			var step string
+			var k kubectl.KCmd
+
+			By("creating a namespace")
+			err := kubectl.CreateNamespace(namespace).ExecV()
+			Expect(err).ToNot(HaveOccurred())
+
+			step = "setting up cass-operator resources via helm chart"
+			ns.HelmInstall("../../charts/cass-operator-chart")
+
+			ns.WaitForOperatorReady()
+
+			step = "creating a datacenter resource with 2 racks/4 nodes and 2 additional seeds"
+			k = kubectl.ApplyFiles(dcYaml)
+			ns.ExecAndLog(step, k)
+
+			ns.WaitForDatacenterReady(dcName)
+
+			checkSeedConstraints()
```

Comment:
I started with a copy of the existing seed integration test.  We can remove these extra validation checks if we want.